jwst/residual_fringe/fitter.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+
+from astropy.modeling.fitting import _model_to_fit_params
+
+
+class ChiSqOutlierRejectionFitter:
+    """Chi squared statistic for outlier rejection"""
+
+    def __init__(self, fitter, domain=None, tolerance=0.0001):
+        self.fitter = fitter
+        self.tolerance = tolerance
+
+        if domain is None:
+            self.domain = 10
```

Comment:
Is `self.domain` ever set to anything other than 10? if not maybe we should drop it.

Reply:
Looking at the old code it seems domain is always set to 10. Should we just define it as self.domain  = 10 in the initialization of the ChiSqOutlierRejectFitter and not even check if it is None

Reply:
There are unit tests planned. Do you have any suggestions for some

Reply:
I added a unit test for the background fitting